lib/tools.go

```diff
@@ -50,19 +55,40 @@ func init() {
 	flag.BoolVar(&ergoTrace, "ergo.trace", false, "enable/disable extended debug info")
```

Comment:
please, update this desc to "enable/disable extended node logging info"

Reply:
Everything else looks good.

Reply:
ok